src/Request/StoriesRequest.php

```diff
@@ -38,6 +39,7 @@ public function __construct(
         public ?FieldCollection $excludeFields = null,
         public ?TagCollection $withTags = null,
         public ?IdCollection $excludeIds = null,
+        public ?RelationCollection $withRelations = null,
```

Comment:
Not sure about the `with` prefix, `relations` would be enough IMHO

Reply:
how would you name it?